distribution/docs/src/main/resources/content/_configuring/logout-theming.adoc

```diff
@@ -0,0 +1,29 @@
+:title: Logout Page Configuration
+:type: configuration
+:status: published
+:parent: Admin Theming Configurations
+:summary: Configure logout pages displayed on ${branding}.
+:order: 01
+
+== {title}
+
+The logout pages is presented to users through the navigation of ${branding} and has a changeable timeout value.
+
+=== Navigating to Logout Configuration
```

Comment:
we probably don't need these section titles, especially if there's only one configuration anyway

Reply:
fixed

Reply:
fixed